test/suite/test_salvage01.py

```diff
@@ -58,42 +58,66 @@ def moreinit(self):
         self.session_params = format
 
         if self.value_format == '8t':
-            # If the test starts failing weirdly, try picking a different byte. Don't forget to
-            # set value_modulus to some smaller value. It had better be < 127 so the mandatory
-            # Python trip through UTF-8 doesn't blow up. Currently it is set to 125, which at
-            # least gets to the salvage code; 126 apparently corrupts the root page and then
-            # nothing works.
-            self.unique = 125
-            self.value_modulus = 113
-            self.uniquebytes = bytes([self.unique])
+            # For FLCS, we write out a sequence of known bytes in adacent rows and search for
+            # that sequence when we want to corrupt the table. This eliminates the problem
+            # earlier versions of this test had where it was looking for a single row value,
+            # namely a single byte, and would sometimes find it in unintended places, such as
+            # the root page, resulting in unsalvageable corruption and tes failure. However,
```

Comment:
LGTM modulo a couple of nitpicks:
* "tes failure" -> "test failure"
* "offets of 8" -> "offsets of 8"
* The logic to set/get the desired value is now longer, so the repetition is a little worse. I think this would benefit from a little refactoring so we could just say `val = self.get_desired_value(i)` or something along those lines.

Reply:
Good thought. Done in [e7cafc7](https://github.com/wiredtiger/wiredtiger/pull/8130/commits/e7cafc767adc130effe3414dd7efb68666b32ec7). (And fixed the typos)

Reply:
Thanks :-)